### ISSUE TITLE
Does it work on a personal PC ?

### ISSUE THREAD

**Issue by RaoHammas** [Dokploy/dokploy#94]
Hi,
I was wondering if this works on a truly self-hosted server. e.g. WSL and then using a domain to route to the self-hosted instance?
I tried it by running it on WSL and then setting up an A Record in DNS pointing to local ipv 4 of server but my domain was unable to successfully open the hosted dokploy server.

**Comment by Mr1Blaze** [comment]
Yes it's works on wls but not anytime. You can use an Virtual Box for more powerful and much more efficient for applications

**Comment by Siumauricio** [comment]
Of course, but only that you will probably have to do extra work to make it work.

**Comment by RaoHammas** [comment]
> Of course, but only that you will probably have to do extra work to make it work.

I tried many things, but domain routing didn't work. I have a domain for which i setup the custom DNS rules which pointed to my local DokPloy server address. In server settings of dokploy i added domain as shown in documentation and YouTube tutorial but it just did not work. I checked DNS was properly configured and was available publicly but for some reason when opening the URL it just said BadRequest with a white page...
